signals/commands/policies/src/main/java/org/eclipse/ditto/signals/commands/policies/modify/ModifyResourceResponse.java

```diff
@@ -132,6 +133,38 @@ public static ModifyResourceResponse created(final PolicyId policyId,
      * @return the response.
      * @throws NullPointerException if any argument is {@code null}.
      * @deprecated Policy ID is now typed. Use
+     * {@link #modified(org.eclipse.ditto.model.policies.PolicyId, org.eclipse.ditto.model.policies.Label, org.eclipse.ditto.model.policies.ResourceKey, org.eclipse.ditto.model.base.headers.DittoHeaders)}
```

Comment:
we could also use the classes here (omitting the package) as they are already imported in the class, this improves readability of the javadoc